run/authentication/auth.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package authentication contains the authentication samples for Cloud Run.
+package authentication
+
+// [START service_to_service_auth]
```

Comment:
The region tag should be in the samples tracker, which requires the region tags to start with the product name. So, this should be something like `run_service_to_service_auth`.

Reply:
Thanks! just created sample number 1692 in the system with that `tag`. Any ideas what's next?